index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+
+
+</head>
+
+<body>
+    <main>
+
+        <header class="header">
+            <a href="#">Marcus</a>
+            <nav><img src="Icons/Union.png" alt="burger icon" /></nav>
+        </header>
+        <section class="content">
+            <div class="greeting-text">
```

Comment:
- [ ] Your headline section currently does not seem to have any Background decoration like in the Figma template as you can see below. Kindly conform to the Figma template as much as possible,
> Your Design
![image](https://user-images.githubusercontent.com/90556221/177000498-d2b95a9a-f1f0-43aa-9021-c2f03af6c334.png)

> Figma Design 
![image](https://user-images.githubusercontent.com/90556221/177000544-38e4ba7e-bf22-4e2b-a880-844d8fe0277b.png)